src/main/java/minicraft/item/TileItem.java

```diff
@@ -62,11 +62,11 @@ protected static ArrayList<Item> getAllInstances() {
 
 		// Creative mode available tiles:
 		items.add(new TileItem("Farmland", SpriteLinker.missingTexture(SpriteType.Item), "farmland", "dirt", "grass", "hole"));
-		items.add(new TileItem("Exploded", SpriteLinker.missingTexture(SpriteType.Item), "explode", "dirt", "grass"));
-		items.add(new TileItem("hole", SpriteLinker.missingTexture(SpriteType.Item), "hole", "dirt", "grass"));
-		items.add(new TileItem("lava", SpriteLinker.missingTexture(SpriteType.Item), "lava", "dirt", "grass", "hole"));
-		items.add(new TileItem("path", SpriteLinker.missingTexture(SpriteType.Item), "path", "dirt", "grass", "hole"));
-		items.add(new TileItem("water", SpriteLinker.missingTexture(SpriteType.Item), "water", "dirt", "grass", "hole"));
+		items.add(new TileItem("Explode", SpriteLinker.missingTexture(SpriteType.Item), "explode", "dirt", "grass"));
```

Comment:
Don't change the name of the exploded tile.

Reply:
The original name of the tile is "explode" and there are no entries depending on the item.

Reply:
Nope, the name before you changed it was Exploded. This will break a lot of saves.

Reply:
Alright, as being saved in inventory, but the name still needed to be corrected.

Reply:
If that breaks saves, we could use mappings.

Reply:
No, just don't change it. Or even better: remove the item.

Reply:
Removing the item just like changing the name, unknown item would be generated.

Reply:
Yes, but the advantage then is that the item doesn't exist.

Reply:
These items are good and conventient for debugging and testing.

Reply:
Then just don't mess with it.

Reply:
Localization keys are required if these are unchanged.

Reply:
Don't touch it.

Reply:
They should be localized, but the original strings do not match the original names (also the localization keys).

Reply:
Just don't change it. You can change the name in loc.

Reply:
Ok.

Reply:
You can change everything else, just not "Exploded".

Reply:
hmm, the tile is called "exploded tile".

Reply:
All the other name changes are no problem. I don't know why you reverted them.

Reply:
For the exploded tile, the tile is indeed called "exploded".

Reply:
Good.